java/code/src/com/suse/manager/reactor/messaging/RegistrationUtils.java

```diff
@@ -148,6 +153,27 @@ public static void finishRegistration(MinionServer minion, Optional<ActivationKe
         if (applyHighstate) {
             MessageQueue.publish(new ApplyStatesEventMessage(minion.getId(), true, emptyList()));
         }
+        // Create a report db user when system is a mgr server
+        if (minion.isMgrServer()) {
+            // create default user with random password
+            MgrServerInfo mgrServerInfo = minion.getMgrServerInfo();
+            Credentials credentials = CredentialsFactory.createCredentials(
+                    "hermes_" + RandomStringUtils.random(8, "abcdefghijklmnopqrstuvwxyz"),
+                    RandomStringUtils.randomAlphanumeric(20),
```

Comment:
if i see this correctly RandomStringUtils is using Random internationally which is not supposed to be used for any security related things like passwords.

Reply:
made a bit different implementation and use SecureRandom()
I think this should be sufficient.